Assets/Scripts/GameManagers/GridManager.cs

```diff
@@ -10,35 +10,34 @@ public class GridManager : MonoBehaviour
     public static Grid grid;
     public static Tilemap tileMap;
 
-    // Constants for the size of the grid
-    public static readonly int MAX_HORIZONTAL = 50;
-    public static readonly int MIN_HORIZONTAL = 0;
-    public static readonly int MAX_VERTICAL = 25;
-    public static readonly int MIN_VERTICAL = 0;
+    // Constants for the size of a level
+    public static List<Level> mapLevels;
```

Comment:
I like that you logically split the single grid/tilemap so we can avoid having to use multiple for the disparate layers.